client/src/modules/home/index.tsx

```diff
@@ -44,6 +45,13 @@ const Home = () => {
       <GridItem colSpan={{ base: 2, xl: 1 }}>
         <VStack align="flex-start">
           <Heading>Upcoming events</Heading>
+          <Link
+            href={`/events/`}
+            _hover={{ color: 'blue' }}
+            textDecoration={'underline'}
+          >
+            View events list
+          </Link>
```

Comment:
I like the extra way to get to `/events`, it's always felt like a chore to have to click 'click for more'.  What do you think to making the heading above into the link?  It's neater, but I'm wondering if it's obvious enough that that's what it does.

Same principle applies to the `/chapters` link.